test/apiMock.js

```diff
@@ -0,0 +1,55 @@
+import fetchMock from 'fetch-mock'
+import faker from 'faker'
+
+function response (data, meta = {}, links = {}) {
+  const jsonApiResponse = {}
+
+  if (data) {
+    jsonApiResponse.data = data
+  }
+
+  if (meta) {
+    jsonApiResponse.meta = meta
+  }
+
+  if (links) {
+    jsonApiResponse.links = links
+  }
+
+  return { body: JSON.stringify(jsonApiResponse) }
+}
+
+function url (path) {
+  return new URL(path, 'http://api/')
+}
+
+export function initTestApi () {
```

Comment:
I'm still not happy with this but it's better than any test-server implementation before.